js/app.js

```diff
@@ -441,13 +532,32 @@ $(document).ready(function () {
     let due_on = milestone.due_on;
     let number = milestone.number;
 
-    let newElementEntry = $('\
-      <div class="milestone-entry ' + uncommittedSignClass + '" ' + action + ' data-number="' + number + '" data-state="' + state + '" data-due_on="' + due_on + '">\
+    let newElementEntry = $(
+      '\
+      <div class="milestone-entry ' +
+        uncommittedSignClass +
+        '" ' +
+        action +
+        ' data-number="' +
+        number +
+        '" data-state="' +
+        state +
+        '" data-due_on="' +
+        due_on +
+        '">\
         <div class="card">\
           <div class="card-body">\
             <div class="flexbox-container">\
-              <input name="title" type="text" class="form-control title-fitting" placeholder="Title" value="' + milestone.title + '" ' + origTitleVal + '>\
-              <input name="description" type="text" class="form-control description-fitting" placeholder="Description" value="' + milestone.description + '" ' + origDescriptionVal + '>\
+              <input name="title" type="text" class="form-control title-fitting" placeholder="Title" value="' +
+        milestone.title +
+        '" ' +
+        origTitleVal +
+        '>\
+              <input name="description" type="text" class="form-control description-fitting" placeholder="Description" value="' +
+        milestone.description +
+        '" ' +
+        origDescriptionVal +
+        '>\
```

Comment:
Personally, I don't think *Prettier* does a good job of formatting HTML code written within a Javascript file. Could we at least address the indentations?

Reply:
This section itself is very messy. We should rewrite this section between back-ticks `

Reply:
But at this point, I don't think it is worth our time to fight again formatting extensions.

Reply:
Are you saying to go ahead with the merge?

Reply:
Yep. I move to go ahead and merge. It's not a bug. It's simply because the formatting extension didn't know how to format the section above.

Reply:
which is odd, because Prettier knows how to format JavaScript and HTML. I think it's because the code section is simply too messy, too long, and too confusing.

Reply:
We'll address this hot mess of a code section in another issue.